doc/cloud-platform_terraform_apply.md

```diff
@@ -0,0 +1,31 @@
+## cloud-platform terraform apply
+
+Terraform apply.
+
+```
+cloud-platform terraform apply [flags]
+```
+
+### Options
+
+```
+      --aws-access-key-id string       Access key id of service account to be used by terraform
+      --aws-region string              [required] aws region to use
+      --aws-secret-access-key string   Secret access key of service account to be used by terraform
+  -h, --help                           help for apply
+      --redact                         Redact the terraform output before printing (default true)
+      --terraform-version string       [optional] the terraform version to use. (default "1.2.5")
+      --workdir string                 [optional] the terraform working directory to perform terraform operation [default] . (default ".")
```

Comment:
This input --workdir seems not used anymore. Can u remove it

Reply:
Can remove, but do you think there might be some use for it testing locally?

Reply:
If that input is not referred anywhere, then it wont be used for testing

Reply:
Removed